docs/platforms/apple/guides/ios/session-replay/performance-overhead.mdx

```diff
@@ -16,7 +16,7 @@ The Pocket Casts app offers a diverse mix of components, including Fragments, Ac
 - **User Flow:** The same flow was executed 10 times to ensure consistency.
 - **Real-World Representation:** This approach closely mirrors performance in real-world scenarios.
 
-The benchmarks were run on an iPhone 14 Pro.
+The benchmarks were run on an iPhone 14 Pro. Note that active Session Replay recording can introduce slow frames on older lower-end iOS devices (for example iPhone 8).
```

Comment:
@jas-kas @bruno-garcia feel free to re-word as necessary

Reply:
lgtm!